recipes/default.rb

```diff
@@ -17,10 +17,22 @@
 # limitations under the License.
 #
 chef_gem "aws-sdk-core" do
-  compile_time false
+  compile_time true
```

Comment:
Just make sure this is the same as in backend-chef, otherwise we could have some weird version conflicts